tools/type_whisperer/api_type_db.proto

```diff
@@ -21,9 +29,13 @@ message TypeDb {
 
   // Map from fully qualified package to the package's name in the next major
   // version of the API.
-  map<string, string> next_version_packages = 2;
+  // map<string, string> next_version_packages = 2;
+
+  // Map from .proto path to the proto's path in the next major version of the
+  // API.
+  // map<string, string> next_version_proto_paths = 3;
```

Comment:
Should we clean these up if no longer in use?

Reply:
done